Source/WebKit/UIProcess/API/Cocoa/_WKWebExtensionContext.mm

```diff
@@ -31,13 +31,15 @@
 #import "_WKWebExtensionContextInternal.h"
 
 #import "CocoaHelpers.h"
+#import "WKWebView.h"
 #import "WebExtension.h"
 #import "WebExtensionContext.h"
 #import "_WKWebExtensionControllerInternal.h"
 #import "_WKWebExtensionInternal.h"
 #import "_WKWebExtensionMatchPatternInternal.h"
 #import "_WKWebExtensionTab.h"
 #import <WebCore/WebCoreObjCExtras.h>
+#import <wtf/URLParser.h>
```

Comment:
Super nit: I feel like one of these sortings must be incorrect, since it has:

w > W, and K > e.